static/sass/craft-editorial.scss

```diff
@@ -0,0 +1,32 @@
+// CRAFT OVERRIDES
+
+.redactor {
+  @extend %gutenberg-component-styles;
+
+  &:after {
+    display: none;
+  }
+
+  p {
+    margin: $font-margin-base;
+  }
+
+  &.dropcap p::first-letter {
+    background: $c-base;
+    color: $c-bg;
+    float: left;
+    font-size: 2.3em;
+    font-weight: $fw-black;
+    line-height: 0.83em;
+    margin: 0.2em 0.4em 0 0;
+    padding: 0.71em;
+  }
+}
+
+cite p {
+  font-size: inherit !important;
+}
+
+.topper-dek {
```

Comment:
again, needed to change this to a `div`, so I added the margins that typically exist on `p` tags.